clients/cobol-lsp-vscode-extension/src/test/suite/lsp.spec.test.ts

```diff
@@ -45,8 +45,9 @@ suite('Integration Test Suite', () => {
     }).timeout(10000).slow(4000);
 
     test('TC152049 Navigate through definitions', async () => {
-        await helper.sleep(500);
+        await helper.sleep(1000);
```

Comment:
why do we need sleep before at the beginning? and why it's longer than before?

Reply:
Hey! Thanks for pointing this out.
I have removed most of the sleep now. It is still needed in some places, as there is some expected delay between the server and client communication.